CHANGELOG.md

```diff
@@ -3,6 +3,59 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.18.1
+_Nov 9, 2023_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- ✨ Multi input date time field section selection
```

Comment:
It's just a tiny bugfix. 🙈 
```suggestion
```

Reply:
you know better than me what is worth mentioning here! :D